### ISSUE TITLE
prism:DialogLayout.CloseOnBackgroundTapped not working

### ISSUE THREAD

**Issue by winsonet** [dansiegel/Prism.Plugin.Popups#148]
I want to use the Prism dialog service in this popups plugin, but when I tried to set the ```CloseOnBackgroundTapped```  and found that's not working (even in your sample project), but I tried other DialogLayout attributes also works fine

Thanks!

**Comment by dansiegel** [comment]
thanks @winsonet I'll take a look at this when I have a free moment to

**Comment by techduggu** [comment]
Hi @dansiegel 
I've also used Prism.Plugin.Popups plugin with Dialog Service and I can add below further information/findings (maybe it helps you during investigation when you get time):

- For me, `prism:DialogLayout.CloseOnBackgroundTapped="True"` is working fine with default position attributes
- If I try to add any Relative position (e.g. `prism:DialogLayout.RelativeHeightRequest="0"` to display dialog at the top), then `CloseOnBackgroundTapped` doesn't close the dialog (basically it doesn't raise the event as I tried to override and explicitly defined prism mask layer (using `<prism:DialogLayout.Mask>`) and handled the TappedEvent - but it didn't trigger the Tap event) 
- Next, I tried using Absolute position (e.g. `prism:DialogLayout.LayoutBounds="0.5, 0, -1, -1"` for top position) and `CloseOnBackgroundTapped` works as expected. - I'm using this as a workaround at present.
- I also noticed that `CloseOnBackgroundTapped` works only when we set `prism:DialogLayout.UseMask="True"` and not False - maybe this is expected here!

Hope the above information helps!
